readme.md

```diff
@@ -183,3 +186,5 @@ Note:
 
 - The test file should be named `index.test-d.ts`.
 - `tsd` supports top-level `await`.
+- When testing promise-returning functions don't use the `await` keyword. Instead, assert directly for a `Promise` like in the example above. When you use `await` your function can potentially return a bare value without being wrapped in a `Promise`, `await` will happily accept non-`Promise` values and your test is rendered meaningless.
+- Use [`const` assertions](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#const-assertions) when you need to pass literal or readonly typed values to functions in your tests.
```

Comment:
I didn't mean this specifically for testing. I meant it for the type definitions themselves.

Reply:
Can you give me an example of how you would use `const` assertions in a definition file?

Reply:
Hmm, you're right. I can't think of any. I always forget how limiting the ambient declaration files are...